src/DonationForms/resources/registrars/templates/fields/File.tsx

```diff
@@ -0,0 +1,20 @@
+import {FieldHasDescriptionProps} from '@givewp/forms/propTypes';
+
+export default function File({Label, ErrorMessage, fieldError, description, inputProps}: FieldHasDescriptionProps) {
+    const FieldDescription = window.givewp.form.templates.layouts.fieldDescription;
+
+    return (
+        <label>
+            <Label />
+            {description && <FieldDescription description={description} />}
+            <input
```

Comment:
Does this input need to be controlled by React Hook Form?

Reply:
Or is it already registered (by means of the `...inputProps`)?

Reply:
Hey @kjohnson, I am actually slated to speak with Jon about the file upload tomorrow. He has been helping us get through how it should be handled. I can provide more details after I've spoken with him.

Reply:
@JoshuaHungDinh following up on this. This looks like the last bit for this PR before we can get it merged.

Reply:
Yea Jon had merged his updated branch with changes to include a hidden field. 21dc20fb02007c2ad04f29f5c0c40a50d581b5c0